### ISSUE TITLE
Accessibility issue with Quantity Selector

### ISSUE THREAD

**Issue by developeryashraj** [storefront-foundation/react-storefront#204]
# Bug report

## Describe the bug

Need to swap `aria-label` for add and substract quantity buttons

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to https://demo.reactstorefront.io/p/3
2. Do inspect on quantity buttons
3. Check the labels are coming wrong currently

## Expected behavior

Plus icon should have aria-label as "add one quantity" and Minus icon should have aria-label as "Substract one quantity"

## Screenshots

![image](https://user-images.githubusercontent.com/3704506/149741130-5d782ba7-ab8d-4bfd-a6ae-56c4c4b5c559.png)

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of React Storefront: [e.g. 7.0.2]
